drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/metadata/ScenarioHeaderMetaData.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
```

Comment:
2018

Reply:
@gitgabrio, please fix all occurrences. :-)

Reply:
And there are several more. For legal reason each one has to be updated to this year.

Reply:
Ok I'll do immediately

Reply:
Fixed with [DROOLS-2672] Create static Scenario grid based on Uberfire Wires Core Grids - Fixed copyright year